aspnetcore/fundamentals/hosted-services/sample/Services/QueuedHostedService.cs

```diff
@@ -44,7 +44,7 @@ public Task StartAsync(CancellationToken cancellationToken)
                 }
                 catch (Exception ex)
                 {
-                    _logger.LogError(ex, "An error occurred executing the workItem.");
+                    _logger.LogError(ex, $"An error occurred executing the {nameof(workItem)}.");
```

Comment:
Does this work? I'm not sure it does.

Reply:
I'll run a test ... I'll add a line to throw inside the `try` and see what it does.

Reply:
Seems ok ...

```
System.NotSupportedException: Geebers McScreebers!
...
BackgroundTasksSample.Services.QueuedHostedService:Error: Error occurred executing workItem.
```